### PR TITLE
Add -XX:+UnlockDiagnosticVMOptions and -XX:+UseAESCTRIntrinsics

### PR DESCRIPTION
**Pull request by radek-starburst**
This PR enables JVM options `-XX:+UnlockDiagnosticVMOptions` and `-XX:+UseAESCTRIntrinsics` to enable intrinsic support for AES CTR/GCM on ARM64. This is motivated by the SSL communication overhead observed with S3.
We observed that it improves performance of network communication with S3 a lot on graviton instances. 

It's enabled in Java 18 by default, which is already released. Therefore risk is minimal.

**Some words of explanation**

In latest Java (OpenJDK) versions the performance of AES CTR/GCM for AARCH64 was significally improved and it was backported to OpenJDK 11 (https://github.com/openjdk/jdk11u-dev/pull/410). The original OpenJDK issue can be found here: https://bugs.openjdk.java.net/browse/JDK-8267993. To use that backport we need to enable it explicitly by
enabling `-XX:+UnlockDiagnosticVMOptions` and `-XX:+UseAESCTRIntrinsics`. It was not enabled by default in the backport to `OpenJDK11` because of conservative approach (as they point in the PR).

### CODE REVIEW

**Comment by hashhar** [comment]
Has this already been merged to Trino itself?

Also you will want to bump the chart version as in https://github.com/trinodb/charts/commit/b50de3135fe53a031ed0caa4cd5a4d9ba0807ba0 so that the new version is released once this is merged.

**Comment by radek-starburst** [comment]
> Has this already been merged to Trino itself?

No, there is a pull request to the Trino (waiting to be merged). Please do not merge this one until I confirm that fact. 
I changed this PR to be a draft.

**Comment by hashhar** [comment]
Merging since the corresponding Trino PR has been merged as https://github.com/trinodb/trino/pull/12251

**Review by hashhar — APPROVED** [review]

**Pull request merged**
